sowingdev/app/curriculum/React.tsx

```diff
@@ -1,15 +1,113 @@
 import React from 'react';
-
+import { CopyBlock, dracula } from 'react-code-blocks';
 interface ReactProps {
   // Define the props for the component here
 }
 
 const ReactComponent: React.FC<ReactProps> = (props) => {
   // Implement the component logic here
 
+    const content = [
+      "Understanding React:",
+      "",
+      "Are you curious about web development and want to understand how modern websites are built? React is a popular JavaScript library that makes building user interfaces easier and more efficient. Here, we'll explore React in simple terms and provide code examples to help you grasp its concepts.",
+      "",
+      "What is React?",
+      "",
+      "React is a JavaScript library for building user interfaces, developed by Facebook. It allows developers to create interactive UI components that can dynamically update as the data changes. React follows a component-based architecture, which means you can break down your UI into reusable pieces.",
+      "To begin using React, you'll need some basic knowledge of HTML, CSS, and JavaScript. Let's start with a simple example:",
+      "",
+      "",
+      "",
+    ];
+
+    const code = `
+// index.html
+  <!DOCTYPE html>
+  <html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>React Example</title>
+    <script src="https://unpkg.com/react@17/umd/react.development.js"></script>
+    <script src="https://unpkg.com/react-dom@17/umd/react-dom.development.js"></script>
+    <script src="https://unpkg.com/babel-standalone@6/babel.min.js"></script>
+  </head>
+  <body>
+    <div id="root"></div>
+      class App extends React.Component {
+      Render() {
+          return (
+            <div>
+              <h1>Hello, React!</h1>
+              <p>Welcome to my first React app.</p>
+            </div>
+          );
+        }
+      }
+      ReactDOM.render(<App>, document.getElementById("root"));
+  </script>
+  </body>
+  </html>
+    `;
+    
+    const content2 = [
```

Comment:
Same thing here right? `content2` is not descriptive enough to understand what we're trying to show.